app/src/common/utils/isEmptyObject.js

```diff
@@ -15,5 +15,8 @@
  */
 
 export function isEmptyObject(obj) {
+  if (typeof obj !== 'object' && obj != null) {
+    throw Error(`argument must be typeOf object`);
+  }
   return Object.keys(obj).length === 0;
```

Comment:
Can you explain this changes?
Also I see this PR https://github.com/reportportal/service-ui/pull/3269
it's in develop branch now but obviously will be in 5.9 too
should we write tests here now?

Reply:
I write tests in this case, because pipeline is failed in case of test coverage less then 60%.
And I chanced this function, because it has incorrect logic  according to it name.
If we pass primitive, for example 1, it returns false, but it is not even an object.

Reply:
Maybe better cherry-pick commit to prevent conflicts in the future
Also function called as `isEmptyObject` why you try put in something else except object?
if you have problem with a data maybe better resolve problem with them
@AmsterGet pls check

Reply:
@Bam6ycha @chivekrodis 
Hope we will use this utility correctly :)
Anyway the Dima's solution more robust and it will indicate wrong behavior in case of unexpected type (f.e. if it is passed through long components chain, backend, etc).
I suggest to merge it as is and provide another PR with changes related to this utility to develop branch.
It's time to plan versions sync activities.